README.md

```diff
@@ -66,3 +102,48 @@ $ kubectl patch workflow -n slurm bb12 --type=json -p '[{"op":"replace", "path":
 
 You can then watch the Workflow resource proceed to Teardown state, after which the burst_buffer.lua teardown function will delete the resource.
 
+### Simple playground exercise #2
+
+This exercise will cause the job to wait on a Major error in DataOut and after
+we clear the error it will proceed to Teardown state where it will wait for
+us to release it.
+
+Edit `testsuite/integration/slurm/jobs/test-bb.sh` to change the `#DW` lines to be the following:
+
+```bash
+#SBATCH --output=/jobs/slurm-%j.out
+#DW DataOut action=error message=johnson_rod_misalignment severity=Major
+#DW Teardown action=wait
+/bin/hostname
+srun -l /bin/hostname
+srun -l /bin/pwd
+```
+
+Submit this job using the `sbatch` command as shown above.
+
+Watch the workflow with `kubectl get workflow -wA` as shown above.
+
+When the workflow is in DataOut state with a status of `TransientCondition`, you can view the error:
+
+```console
+$ kubectl get workflow -n slurm bb3 -o yaml
```

Comment:
```suggestion
kubectl get workflow -n slurm bb3 -o yaml
```

Reply:
Only use the prompt when you need to show output. Also, it's not consistent with the code blocks in the `## Parts of the plugin` section.